charts/event-handler/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for event-handler.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+#
+# Plugin specific options
+#
+teleport:
+  address: ""
+  identitySecretName: ""
+  identitySecretPath: "auth_id"
+
+eventHandler:
+  storagePath: "/var/lib/teleport/plugins/event-handler/storage"
```

Comment:
This has the smell of persistent state, if that's the case, it should probably use a PersistentVolume by default ? Or at a minimum hint at how to configure the volumes/volumemounts to make it so

Reply:
Thanks, you are right, I missed that! I've added a simple example for creating a PVC and a link for more info in the K8s docs.